common/views/themes/typography.ts

```diff
@@ -50,6 +50,7 @@ export const fontSizesAtBreakpoints = {
 };
 
 const fontFamilies = {
+  // TODO rename to reflect Inter?
```

Comment:
👍

Reply:
It's a bigger piece though as it'll affect the whole project, maybe I'll make a new ticket for it.

Reply:
Sounds good. I'd been wondering about font-naming a bit, like should we have e.g. 'sans', 'mono' and 'display' for a bit more abstraction? Only problem there is the next time we change the body font to a serif font, we'll be in trouble with our naming again (maybe unlikely).

I think I have found that having relatively unique strings to describe e.g. `hnr` and `hnb` meant a repo-wide search for those is useful, so we could maybe consider that as part of whatever we update the names to.

Reply:
I think it's useful too! If we're sticking with Inter it'll just be about renaming it across so it's easier to grasp what's used where. As I mentioned earlier today too, @GarethOrmerod discussed adding a font-weight so there might be that too...

Reply:
I think for readability I'll get a PR to just change the names used across first, then worry about the font weights later 😄